src/JoomlaBrowser.php

```diff
@@ -318,4 +318,42 @@ public function uninstallExtension($extensionName)
 		$I->see('There are no extensions installed matching your query.', ['class' => 'alert-no-items']);
 		$this->debug('Extension successfully uninstalled');
 	}
+
+	/**
+	 * Function to Search For an Item in Joomla! Administrator
```

Comment:
Instead of:
` * Function to Search For an Item in Joomla! Administrator`

I would say:
` * Function to Search For an Item in Joomla! Administrator Lists views`

To specify that needs to be used in an admin list